### ISSUE TITLE
Guide me!!!

### ISSUE THREAD

**Issue by BetWithFriend** [shlimp/betwithfriends#16]
קדימה, חייבים לחזור לעבוד...
רוצה להתחיל לבנות הכל מההתחלה?

**Comment by shlimp** [comment]
כן, אני לא מרוצה מהדרך שבה עשינו את זה. ניסיתי משהו חדש ולא אהבתי.
אז עכשיו זה הזמן לנסות עוד משהו חדש
we'll do it in python

we're going to use "Flask" as the web framework, "MySQL" as our database and SQLalchemy as our ORM (communication between python and mysql).

we will still use json rpc as our API.

since I never used Flask or SQLalchemy myself, we both need to do some reading. so dive into python and Flask and SQLalchemy. have fun
